### DISCUSSION TITLE
List of values for various fields?

### DISCUSSION THREAD

**Discussion by idcipher106** [nielsbasjes/yauaa#623] [Q&A] [answered]
We are using this library in our production application and it is working well.  Thank you for releasing it as FOSS!

For some of the key fields, it would be nice to have a list of possible values that the library could return.  For example, what are all the possible values for "DeviceClass"?  What are the possible values for "AgentClass"?

Some others of interest are "DeviceName", "DeviceBrand", "OperatingSystemName", "AgentName".

I realize that in some cases, some of these values might be simple copied from the raw UserAgent string, in which case it is impossible to enumerate all possible values since a device can make anything up.  But for many of these fields, I presume there is a finite list of possible values.

Are these lists of values documented anywhere?
Thank you again for this great project!

**Comment by nielsbasjes** [comment, marked as answer]
There is this page in the documentation: https://yauaa.basjes.nl/expect/fieldvalues/
